jekyll/_cci2/build-project.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: classic-docs
+title: "Build a Project"
+short-title: "Build a Project"
+description: "First project build on CircleCI 2.0"
+categories: [getting-started]
+order: 4
+---
+
+This document describes how to configure your project to run on CircleCI 2.0. 
+
+## Prerequisites
+
+- Complete the steps in the [Join Beta & Sign Up]({{ site.baseurl }}/2.0/first-steps/) document. 
+- Install Docker, see [the Docker store](https://store.docker.com/search?offering=community&type=edition) for a free download  compatible with your operating system. 
+- Download a build image for your language, see [the CircleCI repo on hub.docker.com](https://hub.docker.com/u/circleci/).
```

Comment:
Shouldn't be necessary even for local builds, AFAIK -- I think if you run locally it will download the image on the first run if it sees a new one.